indico/modules/events/timetable/views/__init__.py

```diff
@@ -129,4 +129,6 @@ def inject_meeting_body(event, **kwargs):
 
 def _entry_title_key(entry):
     obj = entry.object
-    return obj.full_title if entry.type == TimetableEntryType.SESSION_BLOCK else obj.title
+    if entry.type == TimetableEntryType.SESSION_BLOCK:
+        return (obj.code, obj.full_title)
```

Comment:
Is this the correct code? Both sessions and session blocks can have a code, and here you use the session block code, but I'm almost certain we use the session code in the conference timetable...

Reply:
I thought we were using program codes to do the sorting, not session codes. I'll check

Reply:
all of them are kind of "program codes" :p

Reply:
Yeah it looks to be sorting by session codes - we should probably update the docs at some point, because they only mention "programme codes" which confused me